cmd/amp/account.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Interactive
+	loginCmd = &cobra.Command{
```

Comment:
Need to change because it is already being used:
```
cmd/amp/login.go:25: loginCmd redeclared in this block
	previous declaration at cmd/amp/account.go:12
```